code/src/components/GameScreen.js

```diff
@@ -0,0 +1,168 @@
+import React from "react";
+import { useDispatch } from "react-redux";
+import { useSelector } from "react-redux";
+import styled from "styled-components/macro";
+import img1 from "../assets/1.jpg";
+import img2 from "../assets/2.jpg";
+import img3 from "../assets/3.jpg";
+import img4 from "../assets/4.jpg";
+import img5 from "../assets/5.jpg";
+import img6 from "../assets/6.jpg";
+
+import { secondFetch } from "../reducers/gameFetch";
+import Loading from "./Loading";
+import EndScreen from "./EndScreen";
+
+const GameScreen = () => {
+  const gameDescription = useSelector((store) => store.gameFetch.description);
+  const gameActions = useSelector((store) => store.gameFetch.actions);
+  const userName = useSelector((store) => store.gameFetch.userName);
+  const loading = useSelector((store) => store.gameFetch.loading);
+  const coordinates = useSelector((store) => store.gameFetch.coordinates);
+
+  const dispatch = useDispatch();
+
+  const onMove = (direction) => {
+    dispatch(secondFetch(userName, direction));
+  };
+
+  return (
+    <>
+      {loading && <Loading />}
+      {coordinates === "1,3" && <EndScreen gameDescription={gameDescription} />}
+
+      {!loading && coordinates !== "1,3" && (
+        <GameContainer coordinates={coordinates}>
+          <Content>
+            <InnerContainer>
+              <GameDescription>{gameDescription}</GameDescription>
+              <DescriptionAndButtonContainer>
+                {gameActions.map((choice, index) => (
+                  <InnerWrap key={index}>
+                    <Description>{choice.description}</Description>
+                    <Button onClick={() => onMove(choice.direction)}>
+                      {choice.direction}
+                    </Button>
+                  </InnerWrap>
+                ))}
+              </DescriptionAndButtonContainer>
+            </InnerContainer>
+          </Content>
+        </GameContainer>
+      )}
+    </>
+  );
+};
+
+export default GameScreen;
+
+const GameContainer = styled.div`
+  background-image: url(${(props) =>
+    props.coordinates === "0,0"
+      ? img1
+      : props.coordinates === "1,0"
+      ? img2
+      : props.coordinates === "1,1"
+      ? img3
+      : props.coordinates === "0,1"
+      ? img4
+      : props.coordinates === "0,2"
+      ? img5
+      : props.coordinates === "0,3"
+      ? img6
+      : "./assets/6.jpg"});
```

Comment:
Generally, if you have long `if-else-if` or nested ternaries you should use `switch`.

In this scenario you could name your images for example `img-0,1` etc and then you just need to:
```jsx
  background-image: url(img-${props.coordinates})
```